src/Resources/config/core.xml

```diff
@@ -49,6 +49,23 @@
             <argument type="service" id="sonata.admin.pool"/>
             <argument type="service" id="sonata.admin.breadcrumbs_builder"/>
         </service>
+        <service id="sonata.admin.controller.crud" class="Sonata\AdminBundle\Controller\CRUDController" public="true">
+            <argument type="service" id="request_stack"/>
+            <argument type="service" id="sonata.admin.pool"/>
+            <argument type="service" id="sonata.admin.global_template_registry"/>
+            <argument type="service" id="sonata.admin.breadcrumbs_builder"/>
+            <argument type="service" id="twig"/>
+            <argument type="service" id="translator"/>
+            <argument type="service" id="session"/>
+            <argument type="service" id="kernel"/>
+            <argument type="service" id="sonata.admin.audit.manager"/>
+            <argument type="service" id="sonata.exporter.exporter" on-invalid="null"/>
+            <argument type="service" id="sonata.admin.admin_exporter" on-invalid="null"/>
+            <argument type="service" id="crsf_token_manager" on-invalid="null"/>
+            <argument type="service" id="logger" on-invalid="null"/>
+            <argument type="service" id="sonata.admin.object.manipulator.acl.admin" on-invalid="null"/>
+            <argument on-invalid="null">%sonata.admin.security.acl_user_manager%</argument>
```

Comment:
I think this argument is not well construct, it should use id as the others and type service, but with id = parameter